src/cli.ts

```diff
@@ -83,14 +86,8 @@ async function run(argv: string[]) {
   }
 
   const fileMap = await generateDeclarations(args.root, config);
-
-  if (args.outDir) {
-    console.log('Writing type declarations to ' + path.resolve(args.outDir));
-    await writeFileMap(args.outDir, fileMap);
-  } else {
-    const concatenated = [...fileMap.values()].join('\n');
-    process.stdout.write(concatenated);
-  }
+  console.log('Writing type declarations to ' + path.resolve(args.outDir));
```

Comment:
Mostly a throwaway suggestion for future console.log instead of space + concatenation try:
```js
console.log('Writing type declarations to', path.resolve(args.outDir))
```

Reply:
Done.